operate/services/health_checker.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2024 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""Source code for checking aea is alive.."""
+import asyncio
+import typing as t
+from concurrent.futures import ThreadPoolExecutor
+
+import aiohttp  # type: ignore
+from aea.helpers.logging import setup_logger
+
+from operate.services.manage import ServiceManager  # type: ignore
+
+
+HTTP_OK = 200
+
+
+class HealthChecker:
+    """Health checker manager."""
+
+    SLEEP_PERIOD = 30
+    PORT_UP_TIMEOUT = 120  # seconds
+
+    def __init__(self, service_manager: ServiceManager) -> None:
+        """Init the healtch checker."""
+        self._jobs: t.Dict[str, asyncio.Task] = {}
+        self.logger = setup_logger(name="operate.health_checker")
+        self.logger.info("[HEALTCHECKER]: created")
+        self._service_manager = service_manager
+
+    def start_for_service(self, service: str) -> None:
+        """Start for a specific service."""
+        self.logger.info(f"[HEALTCHECKER]: Starting healthcheck job for {service}")
```

Comment:
tiny typo, not a big deal 
```suggestion
        self.logger.info(f"[HEALTHCHECKER]: Starting healthcheck job for {service}")
```